doc/Requirements Specification/Parts/Terms and Definitions.lyx

```diff
@@ -735,10 +735,12 @@ newglossaryentry{Palladio}{
 
 \begin_layout Plain Layout
 
-		an approach for the definition of 
+		an approach for the prediction of 
 \backslash
-glspl{component-based software architecture} with a special focus on performance
- properties.
+glspl{component-based software architecture} 
+\backslash
+gls{QoS} with a special focus on performance properties.
+ 
```

Comment:
>an approach for the prediction of component-based software

Shouldn't it be "an approach to predict performance properties of component-based software architecture"?